src/main/resources/liquibase/crop_changelog/14_2_0.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="gelli" id="v14.2.0-1">
+        <preConditions onFail="MARK_RAN">
+            <and>
+                <sqlCheck expectedResult="1">
+                    SELECT CASE WHEN (SELECT COUNT(*) FROM germplsm WHERE methn = 0) > 0 THEN 1 ELSE 0 END;
```

Comment:
We should include in precondition whether there are any records with `methn` not existing in `methods` table.
The first precondition can be simplified/consolidated as:
```
 SELECT COUNT(*) > 0 FROM germplsm 
 WHERE methn = 0 OR NOT EXISTS (
       SELECT 1 FROM methods WHERE mid = methn);
```